### PR TITLE
Fix warnings

### PR DESCRIPTION
**Pull request by sethgho**
Just addressing a couple of warnings.

Change MKAnnotation property to nonatomic copy.
use boolValue when setting isRunningTests.

### CODE REVIEW

File: MetroRappid/CAPStop.h [hunk 1]
```diff
@@ -41,7 +41,7 @@
 #pragma mark - CAPStop MKAnnotation
 
 @property (nonatomic, assign) CLLocationCoordinate2D coordinate;
-@property NSString *title; // FIXME: Do the right thing
-@property NSString *subtitle;
+@property (nonatomic, copy) NSString  *title;
```
Comment (luqmaan) [inline MetroRappid/CAPStop.h]:
``` objc
NSString  *title;
```

One too many spaces after NSString.

**Comment by luqmaan** [comment]
Beautiful. Lets see what travis says.

**Comment by luqmaan** [comment]
Looks like I need to update the cocoapods version or something like that.

**Comment by sethgho** [comment]
Yea I had to update to 0.32 to install the dependencies. I didn't want to
check in that Podfile.lock update though.

On Sat, Apr 19, 2014 at 1:20 PM, Luqmaan Dawoodjee <notifications@github.com

**Comment by luqmaan** [comment]
Just fixed it in #46. Can you pull that and fix the two spaces typo?

**Comment by luqmaan** [comment]
:cat2:

**Comment by luqmaan** [comment]
Thank you!

**Pull request merged**